src/main/java/core/basesyntax/operations/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.FruitTransaction;
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
The class `BalanceOperation` should not start with an empty line. According to the checklist, you should remove all redundant empty lines.